bin/_blackbox_common.sh

```diff
@@ -64,6 +64,15 @@ SECRING="${KEYRINGDIR}/secring.gpg"
 : "${DECRYPT_UMASK:=0022}" ;
 # : ${DECRYPT_UMASK:=o=} ;
 
+# Is this a blackbox repo?
+function is_blackbox_repo() {
+  if [[ -d "$1/keyrings" ]] && [[ -n "$1" ]]; then
```

Comment:
Reverse the order of the tests so that the directory isn't checked if $1 is null:

"""
if  [[ -n "$1" ]] && [[ -d "$1/keyrings" ]]; then
"""